### ISSUE TITLE
A11y_SQL Azure Data Studio_Table_Columns_ScreenReader: The voiceover is not announcing the 'role' of the combo-box of the table data associated with 'Type' column header as combo-box when user use keyboard arrow keys to travel with table data.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19563]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureDataStudiofordesktopMAC_May2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#WCAG4.1.2;#Voiceover;#AINotInScope;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: 1.36.1
Mac Version: macOS Monterey 12.4
Screen Reader: Voiceover

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure Data Studio and connect to server.
3. Tab to 'SERVERS' section and expand.
4. Tab till 'Databases' menu item and expand.
5. Select any of the databases and expand.
6. Tab till 'Tables' sub-menu item.
7. Right click on table sub-menu item and click 'New Table'.
8. Tab till 'Column' tab.
9. Tab till 'int' data type under 'Type' column.
10. Turn on voiceover and observe.

## Actual:
The voiceover is not announcing the 'role' of the combo-box of the table data associated with 'Type' column header as combo-box when user use keyboard arrow keys to travel within table data.

The same issue can be observed for all the tables in the application.

## Expected:
The voiceover should announce the 'role' of the table data associated with 'Type' column header as combo-box when user use keyboard arrow keys to travel within table data. 

## User Impact:
Screen reader users will find it difficult to perceive the user interface component.

## Attachment:
![2_Combobox](https://user-images.githubusercontent.com/98393966/170928204-7206a140-23ef-4aff-845f-42fc7bf478af.gif)

**Comment by bbarik06** [comment]
<img width="1440" alt="19563_Fixed" src="https://user-images.githubusercontent.com/98393966/174241311-9994d412-e72a-464a-8c3d-09b1773c2df7.png">
Issue is not repro now. Hence, closing the bug. Please refer the attachment.

Note: Please do not close the bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
